src/init.c

```diff
@@ -649,7 +649,11 @@ void _julia_init(JL_IMAGE_SEARCH rel)
     jl_prep_sanitizers();
     void *stack_lo, *stack_hi;
     jl_init_stack_limits(1, &stack_lo, &stack_hi);
+#ifdef _OS_EMSCRIPTEN_
```

Comment:
This doesn’t belong here. It should go inside jl_load_dynamic_library with the other platforms.